designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputAge.kt

```diff
@@ -144,11 +145,12 @@ fun InputAge(
                     )
                 }
                 is DateOfBirth, is Age -> {
+                    val inputTextValue = getTextFieldValue(inputType)
                     BasicTextField(
                         modifier = Modifier
                             .testTag("INPUT_AGE_TEXT_FIELD")
                             .fillMaxWidth(),
-                        inputTextValue = TextFieldValue(transformInputText(inputType)),
+                        inputTextValue = inputTextValue,
```

Comment:
isn't it affecting the test?.
Also it would be great to create a snapshot testing of InputAge, what do you think?

Reply:
Now that you mention it I'm actually wondering how those tests didn't fail when we changed to TextFieldValue, I'll add the snapshot test into it